ui/pages/confirm-transaction/confirm-transaction.component.js

```diff
@@ -19,188 +27,191 @@ import {
   ENCRYPTION_PUBLIC_KEY_REQUEST_PATH,
   DEFAULT_ROUTE,
 } from '../../helpers/constants/routes';
+import { isTokenMethodAction } from '../../helpers/utils/transactions.util';
+import { usePrevious } from '../../hooks/usePrevious';
+import {
+  getUnapprovedTransactions,
+  unconfirmedTransactionsListSelector,
+  unconfirmedTransactionsHashSelector,
+} from '../../selectors';
 import {
   disconnectGasFeeEstimatePoller,
+  getContractMethodData,
   getGasFeeEstimatesAndStartPolling,
   addPollingTokenToAppState,
   removePollingTokenFromAppState,
+  setDefaultHomeActiveTabName,
 } from '../../store/actions';
 import ConfirmSignatureRequest from '../confirm-signature-request';
 import ConfirmTokenTransactionSwitch from './confirm-token-transaction-switch';
```

Comment:
Nit: Imports can be organised better.

Reply:
do you have recommendations in mind? 

Slight organization has been done in the follow-up cleanup PR: https://github.com/MetaMask/metamask-extension/pull/17546/